blog/2020-11-05-HowMuchFasterIsJava15.adoc

```diff
@@ -0,0 +1,105 @@
+= How much faster is Java 15?
+:page-interpolate: true
+:awestruct-author: mtomco
+:awestruct-layout: blogPostBase
+:awestruct-tags: [production, benchmark]
```

Comment:
Please add a
`:awestruct-share_image_filename: foo.png`
this is the image that appears in twitter/facebook shares. Just put it the same as your most important image in the text itself.